src/PrettyOrdering.jl

```diff
@@ -0,0 +1,46 @@
+pretty_lt(x, y) = false
+
+pretty_lt(x::Integer, y::Integer) = isless(x, y)
+
+pretty_lt(x::Rational{<:Integer}, y::Rational{<:Integer}) = isless(x, y)
+
+# not sure about the default
+pretty_eq(x, y) = (x == y)
+
+function pretty_lt(x::T, y::T) where {T <: Union{Vector, Tuple}}
+  return pretty_lt_it(x, y)
+end
+
+function pretty_eq(x::Vector, y::Vector)
+  return all(pretty_eq.(x, y))
```

Comment:
this errors in cases of different lengths. Is this desireable, or do we want to return false in these cases instead?